### DISCUSSION TITLE
[SOLVED Pocket integration

### DISCUSSION THREAD

**Discussion by aungkokomm** [martinrotter/rssguard#865] [Ideas]
Do you have any plan to integrate Pocket in near future or any roadmap for Pocket integration? Pocket has been a longest reader for me but I haven't found any good client reader for it. So I do request this feature.

**Comment by martinrotter** [comment]
Hi. I do not know Pocket, but I contacted them to know their position to this subject.

Then we can proceed.

**Reply by martinrotter** [reply]
Pocket responded in a non-positive and bit rude manner actually. They do offer API but at this point the interest from users is non-existent. You are the first one to request Pocket.

If there will be much higher amount of users wanting this, I would implement. Until that PRs are welcomed.

**Comment by ghost** [comment]
I was also looking for pocket integration but I understand development follows user interest.

**Reply by martinrotter** [reply]
Well, I guess it could be added. Their API is relatively simple. Can you write me simple workflow of where/how in RSS Guard should this exactly be integrated? Which menus? Which dialogs etc.?

**Reply by Auskalo** [reply]
Is it possible to add a button to the toolbar that points to an external tool?
The tool would be the browser, and the parameters "https://getpocket.com/edit?url=%1"
